libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -33,63 +56,124 @@ const statesMap: statesMap = {
+              description={formatMessage(
+                mm.reviewScreen.editApplicationModalDesc,
+              )}
+              ariaLabel={formatMessage(
+                mm.reviewScreen.editApplicationModalAria,
+              )}
+              disclosureElement={
+                <Button
+                  colorScheme="default"
+                  iconType="filled"
+                  size="small"
+                  type="button"
+                  variant="text"
+                  icon="pencil"
+                  loading={loadingSubmit}
+                  disabled={loadingSubmit}
+                >
+                  {formatMessage(mm.reviewScreen.buttonsEdit)}
+                </Button>
+              }
+              onConfirm={async () => {
+                const res = await submitApplication({
+                  variables: {
+                    input: {
+                      id: application.id,
+                      event: 'EDIT',
+                      answers: application.answers,
+                    },
+                  },
+                })
+
+                if (res?.data) {
+                  // Takes them back to the editable Review screen
+                  window.location.reload()
+                }
+              }}
+              buttonTextConfirm={formatMessage(
+                mm.reviewScreen.editApplicationModalConfirmButton,
+              )}
+              buttonTextCancel={formatMessage(
+                mm.reviewScreen.editApplicationModalCancelButton,
+              )}
+            />
           </Box>
-        </Text>
+        </Box>
       </Box>
 
-      <Box marginTop={7} marginBottom={8}>
-        <ReviewSection
-          application={application}
-          index={1}
-          state={statesMap['otherParent'][application.state]}
-          title={formatMessage(mm.reviewScreen.otherParentTitle)}
-          description={formatMessage(mm.reviewScreen.otherParentDesc)}
-        />
-        <ReviewSection
-          application={application}
-          index={2}
-          state={statesMap['employer'][application.state]}
-          title={formatMessage(mm.reviewScreen.employerTitle)}
-          description={formatMessage(mm.reviewScreen.employerDesc)}
-        />
-        <ReviewSection
-          application={application}
-          index={3}
-          state={statesMap['vinnumalastofnun'][application.state]}
-          title={formatMessage(mm.reviewScreen.deptTitle)}
-          description={formatMessage(mm.reviewScreen.deptDesc)}
-        />
-      </Box>
+      {(screenState === 'steps' && (
+        <Box marginTop={7} marginBottom={8}>
+          <ReviewSection
+            application={application}
+            index={1}
+            state={statesMap['otherParent'][application.state]}
+            title={formatMessage(mm.reviewScreen.otherParentTitle)}
+            description={formatMessage(mm.reviewScreen.otherParentDesc)}
+          />
```

Comment:
Just as a reminder, at some point this section will need to be conditionally displayed. Or maybe I'm not familiar enough with the business logic, does the other parent always need to approve?

Reply:
Good catch, should be conditional now.